td-agent/yum/td-agent.spec.in

```diff
@@ -162,7 +162,11 @@ fi
 
 %files
 %doc README.md
+%if %{?use_systemd}
```

Comment:
We should use OS version or define another macro since it's not depend on systemd.